src/onegov/activity/collections/invoice.py

```diff
@@ -57,6 +57,13 @@ def for_schema(self, schema, schema_config=None):
         return self.__class__(self.session, self.period_id, self.user_id,
                               schema, schema_config)
 
+    def update_attendee_name(self, attendee_id, attendee_name):
+        invoice_items = self.query_items()
+
+        for item in invoice_items:
+            if item.attendee_id == attendee_id:
+                item.group = attendee_name
+
```

Comment:
This is a little fragile, since we have to remember to use this whenever we change an `attendee_name`. I'd personally prefer adding a trigger/rule on `attendees.name`. But I don't think SQLAlchemy has good support yet for triggers that don't only directly affect a foreign key, so I guess it's fine for now, but maybe add a TODO comment to `Attendee.name` that we would like to trigger automatic updates to any linked `InvoiceItem`s `group` column.